### PR TITLE
gh-1342 always display guard info

### PR DESCRIPTION
**Pull request by sche**
Handles #1342

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
tests are failing

File: Multisig/Logic/Ethereum/GnosisSafe.swift [hunk 1]
```diff
@@ -17,27 +17,8 @@ class GnosisSafe {
         case unknown
     }
 
-    var minimumSupportedVersionValue = "1.0.0"
-    var maximumSupportedVersionValue = "1.3.0"
-    private var minimumSupportedVersion: Version { Version(minimumSupportedVersionValue)! }
-    private var maximumSupportedVersion: Version { Version(maximumSupportedVersionValue)! }
```
Comment (DmitryBespalov) [inline Multisig/Logic/Ethereum/GnosisSafe.swift:23]: I don't understand why this was deleted. the variables were there for testability

Comment (sche) [reply Multisig/Logic/Ethereum/GnosisSafe.swift:23]: True, sorry

File: Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift [hunk 2]
```diff
@@ -198,10 +198,7 @@ class TransactionDetailCellBuilder {
 
             case .setFallbackHandler(let fallbackTx):
                 let handler: Address = fallbackTx.handler.value.address
-                var (label, imageUri) = displayNameAndImageUri(addressInfo: fallbackTx.handler)
-                if label == nil {
-                    label = App.shared.gnosisSafe.fallbackHandlerInfo(AddressInfo(address: handler))?.name ?? "Not set"
```
Comment (DmitryBespalov) [inline Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift:203]: this removed the "Not set" functionality from the tx details, I also don't see where it is now implemented :)

Comment (sche) [reply Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift:203]: So the assumption for "Not set" would be that there is a `setFallbackHandler` tx with zero address?
I would expect the backend to return a label for this case. Or we could return 
```
if label == nil {
  label = handler.isZero ? "Not set" : "Unknown"
}
```
wdyt?

Comment (DmitryBespalov) [reply Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift:203]: yes, check for the label is good

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**